UKHomeOffice/nmsw-ui
# Test UPT status

Update package from convert excel-to-json to node-xlsx
Test UPT status for FAL6
Add test for ETA page info


----

## To test

- npm ci 
- Run with cypress runner

```
npm run cypress:open:dev -- --env MAIL_API_KEY=${MAIL_API_KEY}
```
and choose passenger-status-check.feature
or in cmd line
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/passenger-status-check.feature
```
